### ISSUE TITLE
What about per capita?

### ISSUE THREAD

**Issue by saulshanabrook** [AbeHandler/CovidDataMA#1]
I was checking out [this paper](https://www.nber.org/papers/w27166.pdf) and they do a per capita per county, which maybe would you compare a little closer?

<img width="778" alt="Screen Shot 2020-05-18 at 9 25 37 AM" src="https://user-images.githubusercontent.com/1186124/82218210-8b4b6f00-98e9-11ea-955e-ab7e6ef53138.png">

**Comment by AbeHandler** [comment]
Yeah @kakeith was asking about this too. I think one thing that might be nice is a small indicator of different county population sizes in the table itself. I am slightly wary of per-capita figures b/c there could be other causes of different case volumes. 
![pop](https://user-images.githubusercontent.com/1252925/82224886-6d363c80-98f2-11ea-9f61-b8dafa004465.png)

Thoughts? Would you be willing to put in a PR? I have been meaning to do this for a bit.
